Sming/Libraries/IR/IRremote.cpp

```diff
@@ -27,6 +27,12 @@ IRsend::IRsend(int sendpin)
   irparams.sendpin = sendpin;
 }
 
+//provide backwards compatibility
+IRsend::IRsend()
+{
+  irparams.sendpin = 0;
```

Comment:
In the previous version there is a define:
```
#define IR_OUT_PIN 13 // GPIO13
```
It can be any pin, I will change to 13.